www/src/pages/docs/node-apis-helpers.js

```diff
@@ -0,0 +1,63 @@
+import React from "react"
+import { graphql, Link } from "gatsby"
+import Helmet from "react-helmet"
+import sortBy from "lodash/sortBy"
+
+import APIReference from "../../components/api-reference"
+import { rhythm, scale } from "../../utils/typography"
+import Layout from "../../components/layout"
+import Container from "../../components/container"
+import { itemListDocs } from "../../utils/sidebar/item-list"
+
+class NodeAPIHelperDocs extends React.Component {
+  render() {
+    const docs = sortBy(
+      this.props.data.allDocumentationJs.edges.map(({ node }) => node),
+      docs => docs.name
+    )
+    return (
+      <Layout location={this.props.location} itemList={itemListDocs}>
+        <Container>
+          <Helmet>
+            <title>Node APIs helpers</title>
```

Comment:
I am calling this `helpers` - but those are not really `helpers` - I'm not sure what to call them.

Reply:
This sounds fine. To bike shed for a bit, "Utilities for Node APIs" would work too.

Reply:
I agree Node API helpers is fine here.

```suggestion
            <title>Node API helpers</title>
```